src/js/components/Beacon/BeaconCommon/Filter.tsx

```diff
@@ -47,13 +53,21 @@ const Filter = ({ filter, form, beaconFiltersBySection, removeFilter, isRequired
   const renderLabel = (filter: BeaconFilterUiOptions) => {
```

Comment:
this is effectively a component now - maybe factor it out into a `FilterLabel` component